src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -87,6 +87,9 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
     @Optional
     final MapProperty<String, String> labels = project.objects.mapProperty(String, String)
 
+    @Optional
```

Comment:
Shouldn't this property be annotated with @Internal to indicate that it is neither an input nor an output.

Reply:
Yes I think it should be annotated with @Internal